### ISSUE TITLE
failure to import

### ISSUE THREAD

**Issue by bendichter** [bokeh/ipywidgets_bokeh#2]
```python
from ipywidgets_bokeh import IPyWidget
```

```
---------------------------------------------------------------------------
MultipleInstanceError                     Traceback (most recent call last)
<ipython-input-1-260606c01c5a> in <module>
----> 1 from ipywidgets_bokeh import IPyWidget

~/dev/ipywidgets_bokeh/ipywidgets_bokeh/__init__.py in <module>
----> 1 from .ipy_widget import IPyWidget

~/dev/ipywidgets_bokeh/ipywidgets_bokeh/ipy_widget.py in <module>
     11 from ipywidgets import embed, Widget
     12 
---> 13 from .kernel import kernel
     14 
     15 class IPyWidget(HTMLBox):

~/dev/ipywidgets_bokeh/ipywidgets_bokeh/kernel.py in <module>
     74             self.shell_handlers[msg_type] = getattr(self.comm_manager, msg_type)
     75 
---> 76 kernel = BokehKernel.instance()

~/opt/anaconda3/lib/python3.7/site-packages/traitlets/config/configurable.py in instance(cls, *args, **kwargs)
    421             raise MultipleInstanceError(
    422                 'Multiple incompatible subclass instances of '
--> 423                 '%s are being created.' % cls.__name__
    424             )
    425 

MultipleInstanceError: Multiple incompatible subclass instances of BokehKernel are being created.
```

with release as well as latest

**Comment by ceball** [comment]
@bendichter where are you trying to import - it looks like you're importing in jupyter lab maybe?

The example in the repo shows ipywidgets_bokeh being used to allow ipywidgets in bokeh.

Have you seen https://github.com/bokeh/jupyter_bokeh/?

**Comment by willprice** [comment]
I too had the same issue using bokeh 2.0.1 and ipywidget_bokeh 1.0.0dev1 (both installed from the bokeh channel on conda)

**Comment by mattpap** [comment]
This was fixed in PR #11.
